src/python_testing/matter_testing_infrastructure/matter/testing/runner.py

```diff
@@ -820,11 +820,11 @@ def parse_matter_test_args(argv: Optional[List[str]] = None):
 
     commission_group.add_argument('-m', '--commissioning-method', type=str,
                                   metavar='METHOD_NAME',
-                                  choices=["on-network", "ble-wifi", "ble-thread"],
+                                  choices=["on-network", "ble-wifi", "ble-thread", "nfc-thread"],
```

Comment:
Should we limit only to nfc-thread?
Is the nfc-wifi a option based on the spec?